### PR TITLE
Overlay

### PR DESCRIPTION
**Pull request by grxver7**
Reworking #12, after merging it previously before addressing some flaws.

### CODE REVIEW

File: main/main.gd [hunk 1]
```diff
@@ -16,7 +17,7 @@ func _init():
 
 func _ready():
 	hud.camera_menu_pressed.connect(_on_menu_button_pressed)
-
+
```
Comment (hfxbse) [inline main/main.gd]: Unnecessary whitespace.

File: overlay/ActionMenue.gd
Comment (hfxbse) [inline overlay/ActionMenue.gd:1]: Empty file.

**Review by hfxbse — CHANGES_REQUESTED** [review]
We do not need the generated template and comments from the engine. Please remove them where applicable.

Please follow our naming conversion for files. They should be in `snake_case`. A few of them deserve more descriptive names. For example, `grid_container.gd` could be called `stats_panel.gd` instead.

I am not sure why there are two different settings icons, either one might be left over. In addition, files for the patent view, the dice, and the start menu are in this branch, which should be a separate pull request. Most likely you need to drop f26dc37dda57e58c33684b37eacd867f2df4796b from your branch to get rid of most of them.

Something completely missing is a way to update the panels from the code. Please add methods to the scene which update them. The idea is to later call those from the main loop. In the main script are a few [method templates](https://github.com/hfxbse/mparty/blob/overlay/main/main.gd#L100C12-L100C12) for it already.

After this is done, I'll have a second look and hopefully, we can merge it finally after that :)

**Comment by hfxbse** [comment]
Be aware that 7860cbb7a6a3bab9ee256079bd60c016c7f3e9d9 is a different commit than the commit in the main branch, which is 7366241371419430722022b63d70bd5c50579dab. It is only a cosmetic change, but dropping the first one and rebasing on the latter one might be required.

**Comment by KevinBasener** [comment]
Added methods to update the stats of hud and removed dice and patent view

File: main/main.gd [hunk 2]
```diff
@@ -23,7 +23,7 @@ func _ready():
 		mobile_hud.zoom_out.connect(main_camera.zoom_out)
 
 		hud.add_child(mobile_hud)
-
+
```
Comment (hfxbse) [inline main/main.gd:26]: Unnecessary white space.

**Review by hfxbse — DISMISSED** [review]
* It still does not indicate whose turn it is.
* There are also still two settings icon files. 
* As the patent view button does not do anything, remove it for now. On another note, why have a separate patent view in the first place? Do they not represent something like stars in Mario Party games and therefore be just another value in the stats? It is not like there are types of patents.
* The round indicator should not show zero unless the game has not started yet.
* There are now music settings implemented, no need to show an additional button that does not do anything.
* Not sure how clever it is to show an exit button on a website. What does this even support to do, close the tab? I do not believe this is even possible within Godot. I might be wrong though.

Also, not all elements are properly aligned, but I can accept that for now.

**Comment by grxver7** [comment]
Settings are removed
Patent view also

Players turn was colormarked in the stats menue. Someone removed it

**Comment by hfxbse** [comment]
> Players turn was colormarked in the stats menue. Someone removed it

I know that it was at some point, but it is not now. Could you just add it in again?

**Comment by KevinBasener** [comment]
> > Players turn was colormarked in the stats menue. Someone removed it
> 
> I know that it was at some point, but it is not now. Could you just add it in again?

Is fixed

**Review by ubeF — APPROVED** [review]

**Pull request merged**